<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="dashboard" native="true">
      <property name="styleSheet">
       <string notr="true">#dashboard {
		border-radius: 4px;
        	border: 0.5px solid rgb(180, 180, 180);
		background-color: white;
    }</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="connect_to_iot_button">
         <property name="styleSheet">
          <string notr="true">#connect_to_iot_button {
        	background-color: white; 
		border: 0.5px solid rgb(180, 180, 180);
        	color: black;
		border-radius: 4px
    }
#connect_to_iot_button:hover { 
        background-color: lightgray; 
    }
#connect_to_iot_button:pressed{
		background-color:gray;
	}</string>
         </property>
         <property name="text">
          <string>Push to create connection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="check_connection_button">
         <property name="styleSheet">
          <string notr="true">#check_connection_button {
        	background-color: white; 
		border: 0.5px solid rgb(180, 180, 180);
        	color: black;
		border-radius: 4px
    }
#check_connection_button:hover { 
        background-color: lightgray; 
    }
#check_connection_button:pressed{
		background-color:gray;
	}</string>
         </property>
         <property name="text">
          <string>Push to check connection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="submit_button">
         <property name="styleSheet">
          <string notr="true">#submit_button {
        	background-color: white; 
		border: 0.5px solid rgb(180, 180, 180);
        	color: black;
		border-radius: 4px
    }
#submit_button:hover { 
        background-color: lightgray; 
    }
#submit_button:pressed{
		background-color:gray;
	}</string>
         </property>
         <property name="text">
          <string>submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disconnect_button">
         <property name="styleSheet">
          <string notr="true">#disconnect_button {
        	background-color: white; 
		border: 0.5px solid rgb(180, 180, 180);
        	color: black;
		border-radius: 4px
    }
#disconnect_button:hover { 
        background-color: lightgray; 
    }
#disconnect_button:pressed{
		background-color:gray;
	}</string>
         </property>
         <property name="text">
          <string>Push to disconnect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="random_button">
         <property name="styleSheet">
          <string notr="true">#random_button {
        	background-color: white; 
		border: 0.5px solid rgb(180, 180, 180);
        	color: black;
		border-radius: 4px
    }
#random_button:hover { 
        background-color: lightgray; 
    }
#random_button:pressed{
		background-color:gray;
	}</string>
         </property>
         <property name="text">
          <string>Generate random value</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="panel_window" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="styleSheet">
          <string notr="true">
		border-radius: 4px;
        	border: 0.5px solid rgb(180, 180, 180);
		background-color: white</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_1">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="status_1">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_1">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: black;</string>
            </property>
            <property name="smallDecimalPoint">
             <bool>false</bool>
            </property>
            <property name="value" stdset="0">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_1">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="keyboardTracking">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="device_1_connect_button">
            <property name="styleSheet">
             <string notr="true">#device_1_connect_button {
        	background-color: white; 
		border: 0.5px solid rgb(180, 180, 180);
        	color: black;
		border-radius: 4px
    }
#device_1_connect_button:hover { 
        background-color: lightgray; 
    }
#device_1_connect_button:pressed{
		background-color:gray;
	}</string>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="device_1_disconnect_button">
            <property name="styleSheet">
             <string notr="true">#device_1_disconnect_button {
        	background-color: white; 
		border: 0.5px solid rgb(180, 180, 180);
        	color: black;
		border-radius: 4px
    }
#device_1_disconnect_button:hover { 
        background-color: lightgray; 
    }
#device_1_disconnect_button:pressed{
		background-color:gray;
	}</string>
            </property>
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true">
		border-radius: 4px;
        	border: 0.5px solid rgb(180, 180, 180);
		background-color: white
    </string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="status_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_2">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: black</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="device_2_connect_button">
            <property name="styleSheet">
             <string notr="true">#device_2_connect_button {
        	background-color: white; 
		border: 0.5px solid rgb(180, 180, 180);
        	color: black;
		border-radius: 4px
    }
#device_2_connect_button:hover { 
        background-color: lightgray; 
    }
#device_2_connect_button:pressed{
		background-color:gray;
	}</string>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="device_2_disconnect_button">
            <property name="styleSheet">
             <string notr="true">#device_1_disconnect_button {
        	background-color: white; 
		border: 0.5px solid rgb(180, 180, 180);
        	color: black;
		border-radius: 4px
    }
#device_1_disconnect_button:hover { 
        background-color: lightgray; 
    }
#device_1_disconnect_button:pressed{
		background-color:gray;
	}</string>
            </property>
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="styleSheet">
          <string notr="true">
		border-radius: 4px;
        	border: 0.5px solid rgb(180, 180, 180);
		background-color: white
    </string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="status_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_3">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: black</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="device_3_connect_button">
            <property name="styleSheet">
             <string notr="true">#device_3_connect_button {
        	background-color: white; 
		border: 0.5px solid rgb(180, 180, 180);
        	color: black;
		border-radius: 4px
    }
#device_3_connect_button:hover { 
        background-color: lightgray; 
    }
#device_3_connect_button:pressed{
		background-color:gray;
	}</string>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="device_3_disconnect_button">
            <property name="styleSheet">
             <string notr="true">#device_1_disconnect_button {
        	background-color: white; 
		border: 0.5px solid rgb(180, 180, 180);
        	color: black;
		border-radius: 4px
    }
#device_1_disconnect_button:hover { 
        background-color: lightgray; 
    }
#device_1_disconnect_button:pressed{
		background-color:gray;
	}</string>
            </property>
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="info_board_window" native="true">
      <property name="styleSheet">
       <string notr="true">		border-radius: 4px;
        	border: 0.5px solid rgb(180, 180, 180);
		background-color: white</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="info_board"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
